l10n_cl_etd_xerox/data/etd_document_invoice.xml

```diff
@@ -272,14 +274,14 @@ dte_det_
 {# 31 SubCod #};
 {# 32 FechElabor #};
 {# 33 FechVencim #};
-{# 34 UnmdItem #}{{line.uom_id.name[:4]}};
-{# 35 PrcItem #}{{ line.price_unit|int }};
+{# 34 UnmdItem #}{{ line.uom_id.name[:4] }};
+{# 35 PrcItem #}{{ line.price_unit if tipo != 33 else price_unit_with_tax|int }};
```

Comment:
@dreispt Not sure if we have access to tipo (used in file name) from the content...